src/js/syntax-highlight.js

```diff
@@ -52,10 +52,10 @@ class SyntaxHighlight {
 	* @param {HTMLElement} - The element with a language-relevant class name
 	* @return {String | Null} - The language name e.g. `js` or null if not defined.
 	*/
-	_getLanguage (element) {
-		const className = element.className;
+	_getLanguage(element) {
+		const highlightClassName = [...element.classList].find(c => c.includes('o-syntax-highlight--'));
```

Comment:
```suggestion
		const highlightClassName = element.classList.contains('o-syntax-highlight--');
```
This should work also I think

Reply:
thanks I didn't think about `contains`. That won't work though as `o-syntax-highlight--` is a partial match i.e. to match `o-syntax-highlight--[language]`